pkg/cmd/cli/restic/server.go

```diff
@@ -259,3 +270,83 @@ func (s *resticServer) validatePodVolumesHostPath(client kubernetes.Interface) e
 
 	return nil
 }
+
+// if there is a restarting during the reconciling of pvbs/pvrs/etc, these CRs may be stuck in progress status
+// markInProgressCRsFailed tries to mark the in progress CRs as failed when starting the server to avoid the issue
+func (s *resticServer) markInProgressCRsFailed() {
```

Comment:
Better to use different function name for this function in pkg/cmd/cli/restic/server.go and pkg/cmd/server/server.go